iothub/service/src/DirectMethod/DirectMethodsClient.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Common.Exceptions;
+using Microsoft.Azure.Devices.Http2;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Subclient of <see cref="IotHubServiceClient"/> to directly invoke direct methods on devices and modules in IoT hub.
+    /// </summary>
+    /// <seealso href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-direct-methods"/>
```

Comment:
Reminder to remove the "en-us" part of URLs. This allows the browser to select the right language for the user per their settings, rather than take the one your browser took.